str0m/src/lib.rs

```diff
@@ -1306,6 +1306,7 @@ impl fmt::Debug for MediaData {
             .field("rid", &self.rid)
             .field("time", &self.time)
             .field("len", &self.data.len())
+            .field("packet_range", &self.packet_range())
```

Comment:
`MediaData` is public API, and it's a bit strange to include the `pub(crate)` result here.

Reply:
Good point, do you think we should make `packet_range` public or remove it from debug output?

Reply:
In some respects I want consumers to stop thinking about the RTP layer :) 

So I favor not having it.

Reply:
Aight, I found it very useful for debugging str0m itself but can always add it back on a need by need basis